Documentation/running-fly-stage1.md

```diff
@@ -0,0 +1,77 @@
+# Running rkt with the *fly* stage1
+
+The *fly* stage1 is an alternative stage1 that runs a single-application ACI with only `chroot`-isolation.
+
+
+## How does it work?
+
+In comparison to the default stage1, there is no process manager involved in the stage1.
+
+The rkt application sets up the mounts for `/dev`, `/proc`, `/sys` and the user-provided volumes.
+Then, rkt enters the application's RootFS using the `chroot` syscall and finally executes the application.
+
+Here's a comparison of the default and fly stage1:
+
+stage1-coreos.aci:
+
+```
+host OS
+  └─ rkt
+    └─ systemd-nspawn
+      └─ systemd
+        └─ chroot
+          └─ user-app1
+```
+
+
+stage1-fly.aci:
+
+```
+host OS
+  └─ chroot
+    └─ user-app1
+```
+
+
+## Getting started
+
+You can either use `stage1-fly.aci` from the official release, or build rkt yourself with the right options:
+
+```
+$ ./autogen.sh && ./configure --with-stage1-flavors=fly && make
+```
+
+For more details about configure parameters, see [configure script parameters documentation](build-configure.md).
```

Comment:
Could you update the build-configure.md, by adding fly flavor to the list of flavors listed there?